src/minisketchwrapper.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_MINISKETCHWRAPPER_H
+#define BITCOIN_MINISKETCHWRAPPER_H
+
+#include <minisketch.h>
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
Separate standard library imports:

```suggestion
#include <minisketch.h>

#include <cstddef>
#include <cstdint>
```